patches/ui-base-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
+index 195435b5cc3a6aefc5464f9a5bd8919884783bcb..25349fd0787e0fd9c913fab16d3c646bf856f1f9 100644
+--- a/ui/base/BUILD.gn
++++ b/ui/base/BUILD.gn
+@@ -395,6 +395,7 @@ jumbo_component("base") {
+     "//base:base_static",
+     "//base:i18n",
+     "//base/third_party/dynamic_annotations",
++    "//brave/ui/webui/resources",
```

Comment:
I think adding it here is right.
For the future, how about adding general deps like //brave/ui//base/BUILD.gn`?
It could prevent future deps modification here.
WDYT?

Reply:
@simonhong great idea 😄